packages/p2p/src/pages/my-ads/my-ads-table.jsx

```diff
@@ -6,7 +6,7 @@ import { observer, useStore } from '@deriv/stores';
 import { localize, Localize } from 'Components/i18next';
 import P2pEmpty from 'Components/p2p-empty';
 import ToggleAds from 'Pages/my-ads/toggle-ads.jsx';
-import { TableError } from 'Components/table/table-error.jsx';
+import TableError from 'Components/section-error';
```

Comment:
The name of the component should have the same name as it's component

```suggestion
import SectionError from 'Components/section-error';
```

Reply:
since the sectionerror component is reused in forms and tables, renamed the sections accordingly based on the context.